### PR TITLE
Mythic beasts DNS Component

### PR DESCRIPTION
**Pull request by thinkl33t**
**Description:**

Documentation for Mythic Beasts DNS API

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18333

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/mythicbeastsdns.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: "Mythic Beasts DNS"
+description: "Keep your mythic beasts DNS updated"
+date: 2018-11-09 14:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Network
+featured: false
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/mythicbeastsdns.markdown]: 0.85

File: source/_components/mythicbeastsdns.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+layout: page
+title: "Mythic Beasts DNS"
+description: "Keep your mythic beasts DNS updated"
+date: 2018-11-09 14:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Network
+featured: false
```
Comment (MartinHjelmare) [inline source/_components/mythicbeastsdns.markdown]: This is the default

File: source/_components/mythicbeastsdns.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ha_release: 0.82
+---
```
Comment (MartinHjelmare) [inline source/_components/mythicbeastsdns.markdown:14]: Please add a logo and `ha_iot_class`.

File: source/_components/mythicbeastsdns.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+With the `mythicbeastsdns` component you can automatically update your dynamic DNS entry at [Mythic Beasts](https://www.mythic-beasts.com/).
+
+## {% linkable_title Configuration %}
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+mythicbeastsdns:
+  host: hostname
```
Comment (MartinHjelmare) [inline source/_components/mythicbeastsdns.markdown]: We usually write placeholder values in all caps.

File: source/_components/mythicbeastsdns.markdown [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  password: APIKEYFORMYTHICBEASTS
+```
+
+{% configuration %}
+  host:
+    description: The first part, or subdomain that you want to be dynamic.
+    required: true
+    type: string
+  domain:
+    description: Your TLD (example.com).
```
Comment (MartinHjelmare) [inline source/_components/mythicbeastsdns.markdown]: Please write TLD out. We shouldn't expect everyone to know what that means.

File: source/_components/mythicbeastsdns.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/mythicbeastsdns.markdown]: Resolved.

File: source/_components/mythicbeastsdns.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/mythicbeastsdns.markdown]: Resolved.

File: source/_components/mythicbeastsdns.markdown (hunk 5 above)
Comment (frenck) [reply source/_components/mythicbeastsdns.markdown]: Resolved.

File: source/_components/mythicbeastsdns.markdown (hunk 4 above)
Comment (frenck) [reply source/_components/mythicbeastsdns.markdown]: Resolved.

File: source/_components/mythicbeastsdns.markdown (hunk 3 above)
Comment (frenck) [reply source/_components/mythicbeastsdns.markdown:14]: In his defence, none of the other similar components that do DNS updating, have a `ha_iot_class` defined. At this point, I'm kinda unsure myself on what the `ha_iot_class` should be?

Any suggestion on this @MartinHjelmare?

Comment (ludeeus) [reply source/_components/mythicbeastsdns.markdown:14]: `Cloud Push`?
it does push info out to the cloud.

Comment (frenck) [reply source/_components/mythicbeastsdns.markdown:14]: Done.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for this nice component @thinkl33t! (and of course, thanks for your first contribution to our documentation 👍 )

**Pull request merged**